src/views/UserEdit/UserEdit.js

```diff
@@ -31,6 +31,8 @@ function resourcesLoaded(obj, exceptions = []) {
   return true;
 }
 
+const DEFAULT_FULFILLMENT = 'Hold Shelf';
```

Comment:
you can use the `deliveryFulfillmentValues.HOLD_SHELF` constant here. Also, probably I would make sense to create a  `DEFAULT_FULFILLMENT` constant too,